api/namex/resources/payment/utils.py

```diff
@@ -139,7 +140,33 @@ def merge_payment_request(nr_model, config=None):
                 'country': country,
                 'postalCode': postal_code
             }
-        }
+        },
+        'details': build_payment_details(nr_model)
     }
 
     return payment_request
+
+def build_payment_details(nr_model):
+    """Build payment details."""
+    details = []
+    details.append(
+        {
+            'label': 'NR Number:',
+            'value': nr_model.nrNum
+        }
+    )
+    details.append(
+        {
+            'label': 'Name Choices:',
+            'value': ''
+        }
+    )
+    for idx, name in enumerate(nr_model.names.all(), start=1):
```

Comment:
This looks good, but I think it's possible for the names to come in out of order. It's probably better to put the label as `name.choice + '.'` and then order them after if you need or order the names by the choice beforehand

Reply:
Yes, you're right. If we have the `choice` order number we must use it. :) thanks!